### PR TITLE
add sampling instructions for serverless traces

### PR DESCRIPTION
**Pull request by Hesperide**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/serverless/configuration/_index.md [hunk 1]
```diff
@@ -359,6 +360,12 @@ To parse and transform your logs in Datadog, see documentation for [Datadog log
 
 To see what libraries and frameworks are automatically instrumented by the Datadog APM client, see [Compatibility Requirements for APM][15]. To instrument custom applications, see Datadog's APM guide for [custom instrumentation][16].
 
+## Select sampling rates for ingesting APM spans 
+
+To manage the [APM traced invocation sampling rate][35] for serverless functions, set the `DD_TRACE_SAMPLE_RATE` environment variable on the function you want to change the sampling rate for to a value between 0.000 (trace 0% of Lambda function invocations) and 1.000 (trace all Lambda function invocations).
+
+Setting a sampling rate of 1-10% is recommended to manage usage of traced invocations from your Lambda functions.
```
Comment (tianchu) [inline content/en/serverless/configuration/_index.md:367]: I'm not sure that we really need (or should) give ppl the recommended rate. If we do, probably should give a better (more complex) recommendation, e.g., 100% for cronjob/staging apps, and lower rate for super busy prod functions yielding similar traces.

File: content/en/serverless/configuration/_index.md [hunk 2]
```diff
@@ -359,6 +360,12 @@ To parse and transform your logs in Datadog, see documentation for [Datadog log
 
 To see what libraries and frameworks are automatically instrumented by the Datadog APM client, see [Compatibility Requirements for APM][15]. To instrument custom applications, see Datadog's APM guide for [custom instrumentation][16].
 
+## Select sampling rates for ingesting APM spans 
+
+To manage the [APM traced invocation sampling rate][35] for serverless functions, set the `DD_TRACE_SAMPLE_RATE` environment variable on the function you want to change the sampling rate for to a value between 0.000 (trace 0% of Lambda function invocations) and 1.000 (trace all Lambda function invocations).
```
Comment (tianchu) [inline content/en/serverless/configuration/_index.md:365]: I think we should mention the head-based sampling and you must set the env var on the root service.

Comment (buraizu) [inline content/en/serverless/configuration/_index.md:365]:
```suggestion
To manage the [APM traced invocation sampling rate][35] for serverless functions, set the `DD_TRACE_SAMPLE_RATE` environment variable on the function to a value between 0.000 (trace 0% of Lambda function invocations) and 1.000 (trace all Lambda function invocations).
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Just a small suggestion here to remove some redundant context from a sentence and make it a bit less awkward.

File: content/en/serverless/configuration/_index.md (hunk 1 above)
Comment (Hesperide) [reply content/en/serverless/configuration/_index.md]: good feedback, updated

File: content/en/serverless/configuration/_index.md (hunk 2 above)
Comment (Hesperide) [reply content/en/serverless/configuration/_index.md]: fixed!

Comment (Hesperide) [reply content/en/serverless/configuration/_index.md]: updated to include this

**Review by buraizu — CHANGES_REQUESTED** [review]

**Review by tianchu — APPROVED** [review]

File: content/en/serverless/configuration/_index.md [hunk 3]
```diff
@@ -359,6 +360,18 @@ To parse and transform your logs in Datadog, see documentation for [Datadog log
 
 To see what libraries and frameworks are automatically instrumented by the Datadog APM client, see [Compatibility Requirements for APM][15]. To instrument custom applications, see Datadog's APM guide for [custom instrumentation][16].
 
+## Select sampling rates for ingesting APM spans 
+
+To manage the [APM traced invocation sampling rate][35] for serverless functions, set the `DD_TRACE_SAMPLE_RATE` environment variable on the function to a value between 0.000 (no tracing of Lambda function invocations) and 1.000 (trace all Lambda function invocations).
+
+Metrics are calculated based on 100% of the application’s traffic, and remain accurate regardless of any sampling configuration. 
+
+For high throughput services, there’s usually no need for you to collect every single request as trace data is very repetitive - an important enough problem should always show symptoms in multiple traces. [Ingestion controls][36] help you to have the visibility that you need to troubleshoot problems while remaining within budget.
```
Comment (cswatt) [inline content/en/serverless/configuration/_index.md:369]:
```suggestion
For high throughput services, there’s usually no need for you to collect every single request as trace data is very repetitive—an important enough problem should always show symptoms in multiple traces. [Ingestion controls][36] help you to have the visibility that you need to troubleshoot problems while remaining within budget.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**